examples/src/guides/expression_language/cookbook_function_call.ts

```diff
@@ -27,7 +27,10 @@ const functionSchema = [
 ];
 
 const chain = prompt.pipe(
-  model.bind({ functions: functionSchema, function_call: { name: "joke" } })
+  model.bind({
+    functions: functionSchema,
+    function_call: { name: "joke", arguments: "" },
```

Comment:
I don't think we need to change this right?

Reply:
Thanks, fixed here and elsewhere